crates/ra_hir/src/code_model.rs

```diff
@@ -229,10 +227,10 @@ impl Module {
     }
 }
 
-pub struct Import {
-    pub(crate) parent: Module,
-    pub(crate) id: LocalImportId,
-}
+// pub struct Import {
+//     pub(crate) parent: Module,
+//     pub(crate) id: LocalImportId,
+// }
```

Comment:
@matklad I am not sure it is dead code or not, or for further refactoring ?

Reply:
good catch, this should be removed, as soon as I debug an SO this PR introduced...